ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -18,6 +18,9 @@ parameters:
   required: true
 - name: RELEASE_IMAGE_LATEST
   required: true
+- name: BASE_DOMAIN
+  value: origin-ci-int-aws.dev.rhcloud.com
+  required: true
```

Comment:
I was concerned about this requiring users to set `BASE_DOMAIN`, even if they were fine with the default.  But @chancez pointed me [at][1]:

> Indicates this parameter is required, meaning the user cannot override it with an empty value. If the parameter does not provide a default or generated value, the user must supply a value.

So this PR...

/lgtm

[1]: https://docs.okd.io/latest/dev_guide/templates.html#writing-parameters